lib/delayed_job/delayed_job_callbacks.rb

```diff
@@ -8,20 +8,14 @@ class DelayedJobCallbacks < ::Delayed::Plugin
           block.call(job)
         ensure
           duration = Time.now - start
-          InstrumentalReporters.agent.gauge(job_metric_instrumentation_name(job), duration)
+          InstrumentalReporters.gauge("queue.process", duration) if InstrumentalReporters.configuration[:queue][:process][:enabled]
```

Comment:
We might chain off something similarly to how `Instrumental::Agent` becomes `I`, but as we discussed, in the various pieces that are monkey-patching, defining helper methods on the object being patched is kind of bogus.